RunningOrder/RoundButton.swift

```diff
@@ -29,7 +29,7 @@ struct RoundButton: View {
 
 struct RoundButton_Previews: PreviewProvider {
     static var previews: some View {
-        RoundButton(imageName: NSImage.shareTemplateName, color: Color.green, action: {})
+        RoundButton(image: Image(nsImage: NSImage(named: NSImage.refreshTemplateName)!), color: Color.green, action: {})
```

Comment:
maybe you can create now an extension of image that takes an nsimage name string to beautify this :)